t/05-scheduler-full.t

```diff
@@ -279,7 +296,8 @@ END {
 
 sub dead_workers {
     my $schema = shift;
-    $_->update({t_updated => DateTime->from_epoch(epoch => time - 10200)}) for $schema->resultset("Workers")->all();
+    $_->update({t_updated => DateTime->from_epoch(epoch => time - 3 * 24 * 60 * 60)})
```

Comment:
better we use WORKERS_CHECKER_THRESHOLD + DB_TIMESTAMP_ACCURACY or so or directly set the workers as dead

Reply:
done